condominium/demo/res_company.xml

```diff
@@ -0,0 +1,78 @@
+        <field name="paperformat_id" ref="base.paperformat_us"/>
+        <field name="resource_calendar_id" ref="resource.resource_calendar_std"/>
+        <field name="cash_account_code_prefix">1015</field>
+        <field name="account_folder" ref="documents.documents_finance_folder"/>
+        <field name="currency_provider">ecb</field>
+        <field name="sequence" eval="False"/>
+        <field name="website_id" ref="website.default_website"/>
+        <field name="chart_template">generic_coa</field>
+        <field name="iap_enrich_auto_done" eval="True"/>
+        <field name="documents_spreadsheet_folder_id" ref="documents_spreadsheet.documents_spreadsheet_folder"/>
+        <field name="transfer_account_code_prefix">1017</field>
+        <field name="timesheet_encode_uom_id" ref="uom.product_uom_hour"/>
+        <field name="timesheet_mail_employee_nextdate">2024-03-11 15:47:10</field>
+        <field name="email_primary_color">#000000</field>
+        <field name="anglo_saxon_accounting" eval="True"/>
+        <field name="bank_account_code_prefix">1014</field>
+        <field name="logo_web" type="base64" file="condominium/static/src/binary/res_company/1-logo_web.jpg"/>
+        <field name="logo" type="base64" file="condominium/static/src/binary/res_company/1-logo"/>
+        <field name="account_fiscal_country_id" ref="base.us"/>
+    </record>
+    <record id="res_company_2" model="res.company">
+        <field name="name">Green Island Condominium</field>
+        <field name="user_ids" eval="[Command.link(ref('base.user_admin'))]"/>
+        <field name="partner_id" ref="res_partner_7"/>
+        <field name="documents_hr_folder" ref="documents_hr.documents_hr_folder"/>
+        <field name="project_time_mode_id" ref="uom.product_uom_hour"/>
+        <field name="totals_below_sections" eval="True"/>
+        <field name="paperformat_id" ref="base.paperformat_euro"/>
+        <field name="cash_account_code_prefix">1015</field>
+        <field name="account_folder" ref="documents.documents_finance_folder"/>
+        <field name="currency_provider">ecb</field>
+        <field name="chart_template">generic_coa</field>
+        <field name="iap_enrich_auto_done" eval="True"/>
+        <field name="documents_spreadsheet_folder_id" ref="documents_spreadsheet.documents_spreadsheet_folder"/>
+        <field name="transfer_account_code_prefix">1017</field>
+        <field name="timesheet_encode_uom_id" ref="uom.product_uom_hour"/>
+        <field name="email_primary_color">#000000</field>
+        <field name="anglo_saxon_accounting" eval="True"/>
+        <field name="bank_account_code_prefix">1014</field>
+        <field name="logo_web" type="base64" file="condominium/static/src/binary/res_company/2-logo_web.jpg"/>
+        <field name="logo" type="base64" file="condominium/static/src/binary/res_company/2-logo"/>
+        <field name="account_fiscal_country_id" ref="base.us"/>
+    </record>
```

Comment:
Warnings in the db are annoying. Do you to define all those fields. I quickly checked, and I think that if you only define a few mandatory field (name, user_ids and partner_id should be enought), you don't get those. Could you check?